src/main/java/com/example/BookService/Controller/BookController.java

```diff
@@ -0,0 +1,55 @@
+package com.example.BookService.Controller;
+
+import com.example.BookService.Entity.Book;
+import com.example.BookService.Service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/books")
+public class BookController {
+
+    @Autowired
```

Comment:
Use constructor injection, instead of field/setter injection (@Autowired is not recommended). Why? Check this out - https://betulsahinn.medium.com/why-is-autowired-annotation-not-recommended-4939c46da1f8